### PR TITLE
Add `aws-iam-list-role-policies` command to AWS IAM

### PR DESCRIPTION
**Pull request by amshamah419**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/39651

## Description
Added the new commands"
-  `aws-iam-list-role-policies`
- `aws-iam-get-role-policy`
- `aws-iam-get-policy`

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/AWS-IAM/Integrations/AWS-IAM/README.md [hunk 1]
```diff
@@ -1497,3 +1497,25 @@ Create/update password policy
...
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| roleName | The name of the role to list policies for. | Required | 
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| AWS.IAM.Roles.RoleName.Policies | Unknown | A list of policy names. |
```
Comment (bakatzir) [inline Packs/AWS-IAM/Integrations/AWS-IAM/README.md:1518]: ?
```suggestion
| AWS.IAM.Roles.RoleName.Policies | String | A list of policy names. | 
```

**Review by yuvalbenshalom — COMMENTED** [review]
looks 
please check what possibilities we have for testing

File: Packs/AWS-IAM/Integrations/AWS-IAM/README.md (hunk 1 above)
Comment (amshamah419) [reply Packs/AWS-IAM/Integrations/AWS-IAM/README.md:1518]: It's an array of strings. Still shocked we don't have more typing options yet.

**Comment by amshamah419** [comment]
@yuvalbenshalom Added the commands into the test PB and removed the steps which we cannot run (e.g. create-user)

**Review by yuvalbenshalom — APPROVED** [review]

**Pull request merged**